test/parse_test.rb

```diff
@@ -2053,6 +2053,23 @@ def a(b:
     assert_parses expected, "foo(\n:a,\nb: :c,\n)"
   end
 
+  test "method call with constant paths" do
+    expected = CallNode(
+      ConstantReadNode(CONSTANT("Bundler")),
+      COLON_COLON("::"),
+      IDENTIFIER("uri"),
```

Comment:
This token seems wrong. It should be `CONSTANT("URI")` not the identifier. I think something may have gotten messed up here.